### PR TITLE
Fix migration

### PR DESCRIPTION
**Pull request by ondrajodas**

### CODE REVIEW

File: src/Snowflake/Connection.php [hunk 1]
```diff
@@ -121,29 +125,38 @@ public function assignGrantToRole(array $grant): void
...
+            } else {
+                $this->query(sprintf(
+                    'GRANT %s ON %s %s TO %s %s %s',
+                    $grant['privilege'],
+                    $grant['granted_on'],
+                    $grant['granted_on'] !== 'ACCOUNT' ? $grant['name'] : '',
+                    $grant['granted_to'],
+                    Helper::quoteIdentifier($grant['grantee_name']),
+                    $grant['grant_option'] === 'true' ? 'WITH GRANT OPTION' : '',
+                ));
```
Comment (pivnicek) [inline src/Snowflake/Connection.php]: si myslim ze warehouse granty padnou jen sem kvuli ten `if` nahoru

**Review by pivnicek — DISMISSED** [review]
vypada to fine, jen ze nechceme logovat cele ten query?

File: src/Snowflake/Connection.php [hunk 2]
```diff
@@ -131,32 +131,38 @@ public function assignGrantToRole(array $grant): void
...
         } catch (Throwable $e) {
             if (!$isWarehouseGrant || $this->logger === null) {
                 throw $e;
             }
-            $this->logger->warning($e->getMessage());
+            $this->logger->warning(sprintf(
+                'Failed query "%s" with role "%s"',
+                $query,
+                $grant['granted_by']
+            ));
```
Comment (pivnicek) [inline src/Snowflake/Connection.php]: If we include here $e->getMessage() also then I will be so satisfied :-D

**Review by pivnicek — DISMISSED** [review]
awesome, maybe just include also the error message? ;-)

**Review by pivnicek — APPROVED** [review]

**Review by tomasfejfar — APPROVED** [review]
Formal approval, because OJ and Marc cant :)

**Pull request merged**